modules/nf-core/paragraph/idxdepth/meta.yml

```diff
@@ -0,0 +1,60 @@
+name: "paragraph_idxdepth"
+description: Determines the depth in a BAM/CRAM file
+keywords:
+  - bam
+  - cram
+  - depth
+  - paragraph
+tools:
+  - "paragraph":
+      description: "Graph realignment tools for structural variants"
+      homepage: "https://github.com/Illumina/paragraph"
+      documentation: "https://github.com/Illumina/paragraph"
+      tool_dev_url: "https://github.com/Illumina/paragraph"
+      doi: "https://doi.org/10.1101/635011"
+      licence: "['Apache License 2.0']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
```

Comment:
The other two metas are not documented here. Are they identical? I don't know how we're supposed to deal with that in terms of documentation.

Reply:
Good spot, I'll add them too